Rakefile

```diff
@@ -20,23 +19,30 @@ task default: [:lint, 'test:check']
 namespace :test do
   # run inspec check to verify that the profile is properly configured
   task :check do
-    dir = File.join(File.dirname(__FILE__))
-    sh("bundle exec cinc-auditor check #{dir} --format json --log-level=error | jq .")
+    require 'inspec'
+    puts "Checking profile with InSpec Version: #{Inspec::VERSION}"
+    profile = Inspec::Profile.for_target('.', backend: Inspec::Backend.create(Inspec::Config.mock))
+    pp profile.check
   end
 end
 
-# Automatically generate a changelog for this project. Only loaded if
-# the necessary gem is installed. By default its picking up the version from
-# inspec.yml. You can override that behavior with `rake changelog to=1.2.0`
-begin
+task :changelog do
```

Comment:
the Changelog is generated by a Github action (https://github.com/dev-sec/cis-dil-benchmark/blob/master/.github/workflows/release.yml), there is no need to keep this here

Reply:
Ok, I'll remove it.